### ISSUE TITLE
&'a str ignored

### ISSUE THREAD

**Issue by glademiller** [wafflespeanut/paperclip#74]
I have a struct like 
```rust
#[api_v2_schema]
#[derive(Serialize, Deserialize)]
#[serde(rename_all = "PascalCase")]
pub struct AppVersion<'a> {
    commit: &'a str,
    version: &'a str,
    plan_key: Option<&'a str>,
    build_number: Option<&'a str>,
}
```
in that struct `plan_key` and `build_number` output schema but `commit` and `version` are ignored. `&str` seems like it ought to be supported.

For reference error message:
```
  --> src/service/handlers/util.rs:22:13
   |
22 |     commit: &'a str,
   |             ^

warning: unsupported field type will be ignored.
  --> src/service/handlers/util.rs:23:14
   |
23 |     version: &'a str,
   |      
```

**Comment by wafflespeanut** [comment]
Yeah, we don't support references atm, but they're doable. :smile:

**Comment by wafflespeanut** [comment]
@glademiller Just curious. How are you using that struct in your actix web app?

**Comment by wafflespeanut** [comment]
Fixed in 560a1e4b324abc6ea9da33dc92644d5b6dd67a9f
